.github/workflows/release.yml

```diff
@@ -39,11 +41,48 @@ jobs:
       - name: Build package
         run: python -m build
 
+      - name: Publish to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1.5
+        if: ${{ contains(github.ref, '-rc') }}
+        with:
+          repository_url: https://test.pypi.org/legacy/
+          print_hash: true
+          skip_existing: true
+          verbose: true
+
+      - name: Parse version and output Test PyPI URL
+        id: test-version
+        if: ${{ contains(github.ref, '-rc') }}
+        run: |
+          pip install pyproject.toml
+          output=$(littlepay -v)
+          version=${output/littlepay /}
+          echo "TEST_PYPI_RELEASE_URL=https://test.pypi.org/project/calitp-littlepay/$(version)" > "$GITHUB_OUTPUT"
+
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1.5
+        if: ${{ !contains(github.ref, '-rc') }}
+        with:
+          print_hash: true
+
+      - name: Parse version and output PyPI URL
+        id: version
+        if: ${{ !contains(github.ref, '-rc') }}
+        run: |
+          pip install pyproject.toml
+          output=$(littlepay -v)
+          version=${output/littlepay /}
+          echo "PYPI_RELEASE_URL=https://pypi.org/project/calitp-littlepay/$(version)" > "$GITHUB_OUTPUT"
+
       - name: Release
         uses: softprops/action-gh-release@v1
+        env:
+          PYPI_RELEASE_URL: ${{ steps.version.outputs.PYPI_RELEASE_URL }}
         with:
           files: |
             ./dist/*.tar.gz
             ./dist/*.whl
           prerelease: ${{ contains(github.ref, '-rc') }}
           generate_release_notes: ${{ !contains(github.ref, '-rc') }}
+          append_body: true
```

Comment:
I don't think this line is strictly necessary, since the docs for `generate_release_notes` says:

> ... If body is specified, the body will be pre-pended to the automatically generated notes.

Reply:
New commit [bbbdfe8](https://github.com/cal-itp/littlepay/pull/31/commits/bbbdfe80254c28b907027a88de20a3a9f36d2d4f) does without this line